EqualityComparer/TicketEqualityComparer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AirportTicketBooking.Model.Classes;
+using System.Diagnostics.CodeAnalysis;
+
+namespace AirportTicketBooking.EqualityComparer;
+
+public class TicketEqualityComparer : EqualityComparer<Ticket>
```

Comment:
Why do we need `EqualityComparer` ? Can we override Equals for `Ticket` object?

Reply:
when using LINQ some cases like Contains need this when working with user-defined data types.

Reply:
Got it, but usually in most cases we don't need to check the whole object